### DISCUSSION TITLE
How to monitor Daikin-set-temp in faikin-auto mode?

### DISCUSSION THREAD

**Discussion by turbolooser** [revk/xESP32-Faikout#233] [General]
Hi @revk 
would it be possible to add another sensor which is reporting the actual set-temp of the daikin unit?
So this one?
<img width="445" alt="image" src="https://github.com/revk/ESP32-Faikin/assets/115738363/23691bf3-035b-4e56-ab15-c25d84a1b813">

Thank you

**Comment by revk** [comment]
Ah, OK, well, I used to report this as the target temp, but it confused people, so I changed to report what faikin-auto was aiming for.

An extra sensor for that, hmm, maybe.

**Reply by turbolooser** [reply]
Might be good to monitor if the automation is really working as expected in particular with the advanced parameters for heatback/over etc.

**Reply by turbolooser** [reply]
I have added this now to my configuration yaml to include daikin-target-temp and demand control as entities in HA
Not sure if this could be added to the discovery mqtt of the faikin module

mqtt:
  sensor:
    - name: "Faikin Wohnzimmer Demand Control"
      state_topic: "state/faikin-wohnzimmer/status"
      value_template: "{{ value_json.demand }}"
    - name: "Faikin Wohnzimmer Zieltemperatur Daikin"
      state_topic: "state/faikin-wohnzimmer/status"
      value_template: "{{ value_json.temp }}"
      unit_of_measurement: "°C"
